arm/Microsoft.MachineLearningServices/workspaces/.parameters/parameters.json

```
@@ -17,6 +17,34 @@
         "associatedApplicationInsightsResourceId": {
             "value": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Insights/components/adp-<<namePrefix>>-az-appi-x-001"
         },
+        "systemAssignedIdentity": {
+            "value": true
+        },
+        "computes": {
+            "value": [
+                {
+                    "name": "DefaultCPU",
+                    "location": "westeurope",
+                    "description": "Default CPU Cluster",
+                    "disableLocalAuth": false,
+                    "deploy": true,
+                    "type": "AmlCompute",
```

Lets discuss this point under the suggestion on the deploy.bicep file for the workspaces module where you call the compute module. Come back here if you want to treat these independent from the parent settings.

```suggestion
                    "type": "AmlCompute",
                    "systemAssignedIdentity": true,
                    "userAssignedIdentities": {
                        "/subscriptions/<<subscriptionId>>/resourcegroups/validation-rg/providers/Microsoft.ManagedIdentity/userAssignedIdentities/adp-<<namePrefix>>-az-msi-x-001": {}
                    },
```

---

@MariusStorhaug Updated.